src/Wizard.js

```diff
@@ -98,5 +123,6 @@ const Wizard = ({ children, onComplete, debug }: Props) => {
 
 Wizard.defaultProps = {
   debug: false,
+  stateManager: false,
```

Comment:
Better to not mix data types - `stateManager` is of type `string`. Do you need a defaultProp here? It's already marked as optional by your Flow type.

Reply:
ESlint complains without the defaultProp
<img width="965" alt="Screen Shot 2019-12-11 at 23 03 01" src="https://user-images.githubusercontent.com/31858814/70664313-7697ea80-1c6a-11ea-8af3-b84d6b2760e8.png">

Reply:
Oh ok. Flow-fun! 

Here's what you can do. First,  seal the props:

```
type Props = {|
  onComplete: (currentStep: string) => void,
  children: React$Node,
  debug?: boolean,
  stateManager?: string,
|};
```

Then use `null` as a value for defaultProps :tada:

Reply:
Fixed and force pushed